README.md

```diff
@@ -21,34 +21,48 @@ NAME:
 USAGE:
    auth-thu [options]
    auth-thu [options] auth [auth_options]
+   auth-thu [options] deauth [auth_options]
    auth-thu [options] login
    auth-thu [options] logout
+   auth-thu [options] onlien [online_options]
 
 VERSION:
-
-   1.4
+   1.9
 
 AUTHORS:
-  Yuxiang Zhang <yuxiang.zhang@tuna.tsinghua.edu.cn>
-  Nogeek <ritou11@gmail.com>
+   Yuxiang Zhang <yuxiang.zhang@tuna.tsinghua.edu.cn>
+   Nogeek <ritou11@gmail.com>
+   ZenithalHourlyRate <i@zenithal.me>
 
 COMMANDS:
      auth    (default) Auth via auth4/6.tsinghua
        OPTIONS:
-           --ip value      authenticating for specified IP address
-           --no-check, -n  skip online checking, always send login request
-           --logout, -o    log out of the online account
-           --ipv6, -6      authenticating for IPv6 (auth6.tsinghua)
-           --host value    use customized hostname of srun4000
-           --insecure      use http instead of https
+         --ip value         authenticating for specified IP address
+         --no-check, -n     skip online checking, always send login request
+         --logout, -o       de-auth of the online account (for compatibility)
```

Comment:
"behaves the same as deauth command, for backward-compatibility" would be more clear?

Reply:
applied